Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM openjdk:8-jre
-
-COPY build/install/pdf-service /opt/app/
+FROM openjdk:8-jre-alpine
 
 WORKDIR /opt/app
 
+COPY build/install/pdf-service /opt/app/
+
 HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" curl --silent --fail http://localhost:5500/health
```

Comment:
does curl work here?

Reply:
It probably does not and IT's are no run to detect it :) Will add this after sorting it on the claim-store side.

Reply:
Done